orchestrator/orchestrator.go

```diff
@@ -211,6 +211,10 @@ func (o *Orchestrator) L2Chains() []config.Chain {
 	return chains
 }
 
+func (o *Orchestrator) L2ToL2MessageIndexer() *interop.L2ToL2MessageIndexer {
+	return o.l2ToL2MsgIndexer
+}
+
```

Comment:
see comment on supersim.go

once the admin server start logic is moved into here, we can remove this fn